README.md

```diff
@@ -200,12 +200,7 @@ Using the terraform-plugin-testing framework, each `resource.Test` runs an accep
   - `Check`: function that will verify the state of the resources after the `terraform apply` command has run.
 
 In order to run the full suite of Acceptance tests, run `make testacc`.
-You will also need to set the following environment variables:
-- `ASTRO_API_HOST`
```

Comment:
list keeps changing, dont want to have this info out of date so rather just link to source of truth